packages/ic-http-certification/src/cel/cel_types.rs

```diff
@@ -75,17 +76,35 @@ pub enum DefaultResponseCertification<'a> {
     ///
     /// As many or as little headers can be provided as desired.
     /// Providing an empty list will result in no response headers being certified.
-    CertifiedResponseHeaders(&'a [&'a str]),
+    CertifiedResponseHeaders(Cow<'a, [&'a str]>),
 
     /// A list of response headers to exclude from certification.
     ///
     /// As many or as little headers can be provided as desired.
     /// Providing an empty list will result in all response headers being certified.
-    ResponseHeaderExclusions(&'a [&'a str]),
+    ResponseHeaderExclusions(Cow<'a, [&'a str]>),
+}
+
+impl<'a> DefaultResponseCertification<'a> {
```

Comment:
I added these convenience methods so developers don't have to write `DefaultResponseCertification::CertifiedResponseHeaders(Cow::Borrowed(&headers))` and they can just write `DefaultResponseCertification::certified_response_headers(&headers)`.